### DISCUSSION TITLE
AI Wasserzähler.  Port 3000

### DISCUSSION THREAD

**Discussion by andyunt** [jomjol/AI-on-the-edge-device#2550] [General]
### Issue

In the old Version i can read data with http://server-ip:3000/wasserzaehler.html
Is it possible to check the data in the new Version in the same way ?

I need the data in my Loxone- Miniserver.

Thanks

**Comment by caco3** [comment]
We did some cleanup of the API.
But it is still possible, see here: https://jomjol.github.io/AI-on-the-edge-device-docs/REST-API/#value

Note that we always just provided the web UI and REST API on port 80.

**Reply by andyunt** [reply]
Thanks for the Information.
